bugtool/cmd/configuration.go

```diff
@@ -194,7 +202,13 @@ func defaultCommands(confDir string, cmdDir string, k8sPods []string) []string {
 	commands = append(commands, copyConfigCommands(confDir, k8sPods)...)
 	commands = append(commands, copyCiliumInfoCommands(cmdDir, k8sPods)...)
 
-	return k8sCommands(commands, k8sPods)
+	tcCommands, err := tcInterfaceCommands()
+	if err != nil {
+		return nil, err
```

Comment:
I don't think that this should error out and stop the dumping of all the state just because of a problem with an individual interface. If the bugtool can't dump the set of interfaces, then I suggest that it just prints an error to the commandline and continues.

Reply:
Yeah good point.